layouts/shortcodes/cf-launch.html

```diff
@@ -1,5 +1,5 @@
 <a
-  href="https://console.aws.amazon.com/cloudformation/home?#/stacks/quickcreate?stackName={{ .Get 1 }}&templateURL=https://s3.amazonaws.com/{{ $.Site.Params.template_bucket  }}/templates/{{ $.Site.Params.branch  }}/{{ .Get 0 }}"
+href="https://console.aws.amazon.com/cloudformation/home?#/stacks/create/review?stackName={{ .Get 0 }}&templateURL=https://s3.amazonaws.com/{{ $.Site.Params.template_bucket  }}/templates/{{ $.Site.Params.branch  }}/{{ .Get 1 }}"
```

Comment:
I believe this file is part of an earlier version of the codebase, and this diff is reverting back to the older version. Please remove it from this PR.

Reply:
agreed. confirmed that it's an issue when user is on old version of Cloud Formation console. We will convey that all users must be on the newer version. reverting these changes